File: truyenvn/detail.js
function execute(url) {
    const doc = Http.get(url).html();

    return Response.success({
        name: doc.select("h1.name").first().text(),
        cover: doc.select(".book img").first().attr("data-src"),
        author: doc.select(".author a").first().text(),
        description: doc.select(".comic-description").html(),
        detail: doc.select(".meta-data").html(),
        host: "https://truyenvn.com",
        ongoing: doc.select(".status").text().indexOf("Đang Cập Nhật") >= 0
    });
}

File: medoctruyentranh/detail.lua
local url = ...
local doc = http:get(url):html()
if doc ~= nil then
    local info = doc:select(".detail_infos")

    local book = {
        ["name"] = info:select(".title"):text(),
        ["cover"] = doc:select(".detail_info img"):first():attr("src"),
        ["host"] = "https://www.medoctruyentranh.net",
        ["author"] = info:select(".other_infos font"):first():text(),
        ["description"] = info:select(".summary"):first():html(),
        ["detail"] = info:select(".other_infos"):last():html(),
        ["ongoing"] = not text:contains(info:html(), "Đã kết thúc"),
        ["url"] = text:replace(url, "m.medoctruyentranh.net", "www.medoctruyentranh.net")
    }
    return response:success(book)
end
return nil